tox.ini

```diff
@@ -95,7 +94,7 @@ deps =
     #flake8-quotes
 
 commands =
-    - isort --check-only --recursive --diff {toxinidir}/src setup.py
+    isort --check-only --diff {toxinidir}/src setup.py
     flake8 src docs setup.py
```

Comment:
Just a remark - I removed the `-` - which does not make any difference, as using the `diff` flag makes isort to always return a return code of 0 - and the `-` sign is used just in case when the return code is not zero, which cannot happen here anyway.